images-list

```diff
@@ -311,10 +311,12 @@ openpolicyagent/gatekeeper rancher/mirrored-openpolicyagent-gatekeeper v3.3.0
 openpolicyagent/gatekeeper rancher/mirrored-openpolicyagent-gatekeeper v3.4.0
 openpolicyagent/gatekeeper rancher/mirrored-openpolicyagent-gatekeeper v3.5.1
 openpolicyagent/gatekeeper rancher/mirrored-openpolicyagent-gatekeeper v3.5.2
+openpolicyagent/gatekeeper rancher/mirrored-openpolicyagent-gatekeeper v3.6.0
 openpolicyagent/gatekeeper rancher/openpolicyagent-gatekeeper v3.1.0
 openpolicyagent/gatekeeper rancher/openpolicyagent-gatekeeper v3.1.1
 openpolicyagent/gatekeeper rancher/openpolicyagent-gatekeeper v3.2.1
 openpolicyagent/gatekeeper rancher/openpolicyagent-gatekeeper v3.3.0
+openpolicyagent/gatekeeper-crds rancher/mirrored-openpolicyagent-gatekeeper-crds v3.6.0
```

Comment:
@PennyScissors do we need the crds here ? we haven't mirrored that before.

Reply:
Gatekeeper upstream started adding crd image in `values.yaml` from `v3.6.0` you can find it [here](https://github.com/open-policy-agent/gatekeeper/blob/19ee221352a4d34ea659512e5ab376bae5fc17b8/charts/gatekeeper/values.yaml#L28).
In earlier versions there was only one image. For example if you check it for `v3.5.1` there was only the gatekeeper image. It's the [permalink](https://github.com/open-policy-agent/gatekeeper/blob/c8621550a3cc38d11f08b33a5c2b74a9ad328b1f/charts/gatekeeper/values.yaml#L24) for reference.

Reply:
Also a new repo has been created for crds. Here is the issue. https://github.com/rancherlabs/eio/issues/563

Reply:
@cbron That's correct, upstream added this new image in v3.6.0 so we need to mirror it.
![image](https://user-images.githubusercontent.com/8951271/131880181-dcb971b2-0a6d-4b3a-b115-fa6071ad1495.png)